v2/pkg/release/signature.go

```diff
@@ -25,10 +25,26 @@ type SignatureSchema struct {
 	Log    clog.PluggableLoggerInterface
 	Config v1alpha2.ImageSetConfiguration
 	Opts   mirror.CopyOptions
+	pgpKey string
 }
 
 func NewSignatureClient(log clog.PluggableLoggerInterface, config v1alpha2.ImageSetConfiguration, opts mirror.CopyOptions) SignatureInterface {
-	return &SignatureSchema{Log: log, Config: config, Opts: opts}
+	var pgp string
+	if pgpKeyOverride := os.Getenv("OCP_SIGNATURE_VERIFICATION_PK"); len(pgpKeyOverride) != 0 {
+		log.Info("OCP_SIGNATURE_VERIFICATION_PK environment variable set: using PGP key in %s for OCP signature verification", pgpKeyOverride)
```

Comment:
From my perspective oc-mirror should show only high level information as Info, signature details is something someone is going to verify only when there is something wrong (Debug Level).